### ISSUE TITLE
Battle Chess 2 - Chinese Chess (1990)

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoDOS#5340]
Bobinator reported the issue from the screenshot at the bottom of the ticket and I confirmed it. Happens at a random time during gameplay. I tried loadfix but still got the error. I then tried the below and got through two games so this may be the solution.

[dos]
xms=false
ems=false
umb=false

![image](https://github.com/user-attachments/assets/921f7665-5935-4484-9c9d-ec8fc1294f9c)

**Comment by exoscoriae** [comment]
changes pushed

**Comment by Python-Exoproject** [comment]
I think you may have made the change to `Battle Chess (1988)` instead of `Battle Chess 2 - Chinese Chess (1990)` as I dont see the change in Battle Chess 2 and Battle Chess 1 has been updated yesterday and has the requested values. I dont think its affecting Battle Chess 1, but changes still need to be made to Battle Chess 2

**Comment by exoscoriae** [comment]
good catch

**Comment by Python-Exoproject** [comment]
Played another few games and while it seems less frequent the issue still occurs. Any thoughts? Ive tried loadfix and reducing memsize



![Image](https://github.com/user-attachments/assets/6c33abfe-49ed-42ed-9814-018eb8490cf1)

**Comment by SmilingSpectre** [comment]
Maybe remove 100% cycles, and try something more fixed? Maybe coincidence, but I didn't try max cycles.

**Comment by Python-Exoproject** [comment]
Yeah that and I think a higher loadfix value helps too. I tried `loadfix -100` and completed two games

**Comment by exoscoriae** [comment]
i don't like these fixes. There is no science behind them. You said prior that you played two games with no issue as well.

**Comment by exoscoriae** [comment]
cycles set to 5k.

**Comment by Python-Exoproject** [comment]
Thats the nature of a random issue. At least these changes are safe ones or I wouldnt suggest making them. At worst they wont fix the issue
